java/ql/src/utils/model-generator/ModelGeneratorUtils.qll

```diff
@@ -151,11 +151,29 @@ predicate isRelevantTaintStep(DataFlow::Node node1, DataFlow::Node node2) {
   )
 }
 
-string returnNodeAsOutput(TargetAPI api, ReturnNodeExt node) {
+predicate isRelevantContent(DataFlow::Content f) {
+  isRelevantType(f.(DataFlow::FieldContent).getField().getType()) or
```

Comment:
Should synthetic fields also be covered?

Reply:
Yes, you are right. Will add it.